humilis_sam/outputs.yaml.j2

```diff
@@ -17,7 +31,7 @@ outputs:
                 - - 'https://'
                   - Ref:
                       {% if swagger_template %}
-                      ApiGateway{{_layer.name}}{{_env.stage}}
```

Comment:
Why would we need to add the layer name and the stage to the logical name of the resource? Makes the resulting CF template uglier and both the layer name and the stage are already present in the name of the associated CF stack.